README.md

```diff
@@ -148,6 +148,14 @@ plainwhite:
 
 Search is powered by [Simple-Jekyll-Search](https://github.com/christian-fei/Simple-Jekyll-Search) Jekyll plugin. A `search.json` containing post meta and contents will be generated in site root folder. Plugin JavaScript will then match for posts based on user input. More info and `search.json` customization documentation can be found in plugin repository.
 
+**Base URL**
+
+You can specify a custom base URL (eg. example.com/blog/) by adding the following line to config.yaml. Note that there is no trailing slash on the URL.
```

Comment:
Don't you think it'll be better if we change it to `_config.yml`, the exact name so that people don't get confused?

Reply:
Good catch, I'll change it now.